src/service/message.js

```diff
@@ -24,62 +49,89 @@ const findAll = () => {
   return [];
 };
 
-const findOne = id => {
+const findOne = messageId => new Promise((resolve, reject) => {
   if (fs.existsSync('log.txt')) {
-    const fileData = fs.readFileSync('log.txt', 'utf8');
+    const rl = createInterface();
 
-    if (fileData) {
-      const message = fileData.split('\n')[id];
+    rl.on('line', line => {
+      if (line) {
+        const { id, message } = JSON.parse(line);
 
-      if (message) {
-        return message;
+        if (id === parseInt(messageId, 10)) {
+          resolve(message);
+          rl.close();
+          rl.removeAllListeners();
+        }
       }
-    }
-  }
+    });
 
-  return null;
-};
+    rl.on('close', () => {
+      reject(new Error('Message has not been found'));
+    });
+  } else {
+    reject(new Error('file not exiest'));
```

Comment:
`Message has not been found` should be thrown. User doesn't care if file doesn't exist.